be/src/formats/parquet/parquet_cli_reader.h

```diff
@@ -0,0 +1,256 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <exception>
+#include <filesystem>
+
+#include "common/status.h"
+#include "formats/parquet/file_reader.h"
+#include "fs/fs.h"
+#include "runtime/descriptor_helper.h"
+
+namespace starrocks::parquet {
+
+class ParquetCLIReader {
+public:
+    ParquetCLIReader(const std::string& filepath)
```

Comment:
What is this class for? Unit testing?

Reply:
Yes, of course we can implement an parquet cli reader later, just like `parquet-tools`